cfgov/jinja2/v1/_includes/templates/header.html

```diff
@@ -1,20 +1,20 @@
-<header class="o-header
-               header"
+<header class="o-header"
```

Comment:
Yup, good catch. Fixed in https://github.com/cfpb/cfgov-refresh/commit/7321c8b8d227912d596290a19a8c2ce47eac0ca4